week3/ml_decision_tree_sample/unit_tests/test_model.py

```diff
@@ -0,0 +1,37 @@
+import sys, os
+import pytest
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname('ml_decision_tree_sample'), '..')))
+from predictor import model_for_predict
+from prepare_data import prepare_data
+from generator_model_card import generate_model_card
+
+
+def test_overfit_batch():
+    clf = model_for_predict(max_depth=11, min_samples_leaf=3, min_samples_split=2,
+                            n_estimators=30,
+                            random_state=42, patch_to_file='short_test_data.csv')
+    assert (clf.predict(prepare_data('short_test_data.csv')[0]['x']) == prepare_data('short_test_data.csv')[0][
+        'y'].to_numpy()).all(), 'model not work'
```

Comment:
what dose this mean? not sure I will understand this code after couple of months

Reply:
chenged